database/query/profile_status.sql

```diff
@@ -137,21 +138,23 @@ SELECT
     rt.id AS rule_type_id,
     rt.guidance as rule_type_guidance,
     rt.display_name as rule_type_display_name
-FROM rule_evaluations res
```

Comment:
In the evaluation history tables, we are guaranteed that there will always be one alert and remediation row per evaluation status, and at least one evaluation status and latest evaluation status per evaluation rule entity. This is because the data is inserted transactionally into the DB.